### PR TITLE
teams fix

### PR DESCRIPTION
**Pull request by cannedyeti**
Frick @brycerobinsn @MichaelDulin  and @abeck3423 

also @trooper5010

### CODE REVIEW

**Comment by cannedyeti** [comment]
There is an error here. Find it.

File: commands/teams.js [hunk 1]
```diff
@@ -16,19 +16,36 @@ function createTeamsEmbed(gamerArray, rerolledBy) {
 }
 
 async function generateTeams(interaction, rerolledBy) {
-  let channel = interaction.member.voice.channel;
+  const channel = interaction.member.voice.channel;
+  let excudeArr = []
```
Comment (brycerobinsn) [inline commands/teams.js:20]:
```suggestion
  let excludeArr = []
```

File: commands/teams.js [hunk 2]
```diff
@@ -16,19 +16,36 @@ function createTeamsEmbed(gamerArray, rerolledBy) {
...
+  if(!rerolledBy) {
+    excudeArr = [
```
Comment (brycerobinsn) [inline commands/teams.js:22]:
```suggestion
    excludeArr = [
```

File: commands/teams.js [hunk 3]
```diff
@@ -16,19 +16,36 @@ function createTeamsEmbed(gamerArray, rerolledBy) {
...
+  }
   if(!channel) {
     await interaction.reply("You must be in a voice chat to create teams.")
   } else {
     let membersInVoiceChannel = [];
-    // let membersInVoiceChannel = ['one','two','three','four','five','six','seven','eight', 'nine'];
+    // let membersInVoiceChannel = ['one','two','three','four','five','six'];
     channel.members.forEach(({ user }) => {
-      membersInVoiceChannel.push(user.username)
+      if(!excudeArr.includes(user.username)) {
```
Comment (brycerobinsn) [inline commands/teams.js:36]:
```suggestion
      if(!excludeArr.includes(user.username)) {
```

**Review by brycerobinsn — CHANGES_REQUESTED** [review]
lrn2spell

**Pull request merged**